openshift/machine-config-operator
# Bug 1798788: Set Kubelet node IP to non-vip

This change prevents:
* Kubelet choosing as node IP an address from a non control plane subnet
  (like the provisioning network)
* Kubelet choosing a deprecated IPv6 address as its node IP that in
  several platform can be a VIP

**- How to verify it**

Deploy on an IPv6 environment on a platform that uses keepalived managed Virtual IPs for its API, DNS and Ingress and see that the node IP is none of those, as those are created by keepalived as deprecated addresses.
